mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB="$(mktemp --suffix -snapshot.db)"
+SNAPSHOT_SETTINGS_PATH="$(mktemp --suffix -snapshot_settings.py)"
```

Comment:
Indeed, I get this error:

```
+ ./manage.py loaddata /tmp/tmp.8y7acV4xV6-snapshot.json.gz --settings=/tmp/tmp.4l7WYTRLQl-snapshot_settings.py
Traceback (most recent call last):
  File "./manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/home/deploy/milestone-a/vendor/packages/Django/django/core/management/__init__.py", line 453, in execute_from_command_line
    utility.execute()
  File "/home/deploy/milestone-a/vendor/packages/Django/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/deploy/milestone-a/vendor/packages/Django/django/core/management/__init__.py", line 263, in fetch_command
    app_name = get_commands()[subcommand]
  File "/home/deploy/milestone-a/vendor/packages/Django/django/core/management/__init__.py", line 109, in get_commands
    apps = settings.INSTALLED_APPS
  File "/home/deploy/milestone-a/vendor/packages/Django/django/conf/__init__.py", line 53, in __getattr__
    self._setup(name)
  File "/home/deploy/milestone-a/vendor/packages/Django/django/conf/__init__.py", line 48, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/deploy/milestone-a/vendor/packages/Django/django/conf/__init__.py", line 134, in __init__
    raise ImportError("Could not import settings '%s' (Is it on sys.path?): %s" % (self.SETTINGS_MODULE, e))
ImportError: Could not import settings '/tmp/tmp.4l7WYTRLQl-snapshot_settings.py' (Is it on sys.path?): Import by filename is not supported.
```

@onceuponatimeforever I'll have to revert this; please te-test it and resubmit! I'm excited about getting this feature working.